src/System.Linq/tests/SelectTests.cs

```diff
@@ -139,8 +139,8 @@ public void SelectPropertyPassingIndexOnLast()
             Assert.Equal(expected, source.Select((e, i) => i == 5 ? e.name : null));
         }
 
-        [Fact]
-        [ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
+        [Fact(Skip = "Valid test but too intensive to enable even in OuterLoop"]
```

Comment:
@Jiayili1 I may mave missed this from the steps, but before submitting a PR it's wise to build the tree from the root with "build.cmd". You can also build an individual project with "msbuild <projectname>.csproj".

The CI results here will show the build errors, but it takes a lot of time to get them back when building locally would find issues in a moment.